tests/rbd_mirror/test_snapshot_mirroring_on_pool_negative.py

```diff
@@ -0,0 +1,61 @@
+from ceph.rbd.initial_config import initial_mirror_config
+from ceph.rbd.workflows.cleanup import cleanup
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """Verify that snapshot based mirroring cannot be enabled at pool level
+    Pre-requisites :
+    We need atleast one client node with ceph-common, fio and rbd-nbd packages,
+    conf and keyring files in both clusters with snapshot based RBD mirroring
+    enabled between the clusters.
+    Test cases covered -
+    1) CEPH-83573617 - [negative scenario] Verify snap mirroring on pool level
+    kw:
+        config:
+            rep_pool_config:
+              num_pools: 1
+              num_images: 1 # one each of below encryptions will be applied on each image
+              mode: pool
+              mirrormode: snapshot
+            ec_pool_config:
+              num_pools: 1
+              num_images: 1 # one each of below encryptions will be applied on each image
+              mode: pool
+              mirrormode: snapshot
+
+    Test Case Flow
+    1. Bootstrap two CEPH clusters and create a pool and an image each for replicated and ec pools
+    2. Setup snapshot based mirroring in between these clusters at pool level
+    3. Verify that enabling snapshot based mirroring at pool level errors out appropriately
+    """
+    pool_types = ["rep_pool_config", "ec_pool_config"]
+    log.info(
+        "Running negative scenario of enabling snapshot based mirroring at pool level"
+    )
+    try:
+        ret_val = 0
+        kw.get("config", {})["do_not_run_io"] = True
+        mirror_obj = initial_mirror_config(**kw)
```

Comment:
can we mention `log.info` for executing test on Replicated pool and EC pool how we did other test?

Reply:
Since the complete test is being executed in one go inside the initial_mirror_config method itself, that log can't be written. However when the test is complete, there is a log which says Snapshot mirroring cannot be enabled on pool mode for both rep pool and ec pool separately. I think this log should be sufficient.

Reply:
i am saying like this way https://github.com/red-hat-storage/cephci/blob/2faeceac9b6611f023511c0bfef12d9db9880928/tests/rbd_mirror/test_mirror_move_secondary_trash.py#L75